pylint_django/tests/test_func.py

```diff
@@ -6,6 +6,8 @@
 import pylint
 
 
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'pylint_django.tests.conftest')
+
```

Comment:
Why do we need this? checkers/django_installed.py also tries to import Django and it used to work without anything else so please explain.

Reply:
I added this so that the test configuration would be used when `django.setup` is called in `_module_name_from_django_model_resolution` (https://github.com/PyCQA/pylint-django/pull/272/files#diff-1bbed731b242312a86f3f314c3682343R47). But looking back on it, I think it'd be better to call `settings.configure` before `django.setup` to avoid messing with environment variables.

Reply:
Makes sense then. Let's name this module settings.py though.